br/tests/br_key_locked/codec.go

```diff
@@ -63,7 +63,8 @@ func (c *codecPDClient) ScanRegions(
 		endKey = codec.EncodeBytes(nil, endKey)
 	}
 
-	regions, err := c.Client.BatchScanRegions(ctx, []pd.KeyRange{{StartKey: startKey, EndKey: endKey}}, limit, opts...)
+	//nolint:staticcheck
+	regions, err := c.Client.ScanRegions(ctx, startKey, endKey, limit, opts...)
```

Comment:
those can be removed if we have do it in pd `client-go`?

Reply:
Sorry I don't get your mean, I just revert it to the old version.

Reply:
i mean if we have make interface compatible in `client-go` side, we can still use `BatchScanRegions` here

Reply:
It's a little weird that pd's client handle unimplement error, and the compatibility is made out of pd client. So I switch back to old interface here.